src/main/java/com/st/utopia/cancellationservice/model/Ticket.java

```diff
@@ -0,0 +1,103 @@
+package com.st.utopia.cancellationservice.model;
+
+import java.time.LocalDateTime;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Table;
+
+/**
+ * An airport model.
+ *
+ * @author Henry Cho
+ * 
+ * Model used for the purpose of the cancellation microservice.
+ * 
+ * There are no set methods for information about the ticket such as flight number, row/seat, or class.
+ * 
+ * Upon cancelling the ticket, the microservice needs to set the following fields as null:
+ * 	-reserver
+ * 	-price
+ * 	-reservation_timeout
+ * 	-booking_id
+ * 
+ * This model does not need to define a composite key since the cancellation microservice will search for the ticket via booking_id which is unique.
+ */ 
+
+@Entity
+@Table(name="tbl_tickets")
+public class Ticket{
+	@Column(name="flight")
+	Integer flight;
+	@Column(name="row")
+	Integer row;
+	@Column(name="seat")
+	Character seat;
+	@Column(name="class")
+	Integer tier; //class is a keyword =(
+	@Column(name="reserver")
+	Integer reserver;
+	@Column(name="price")
+	Integer price;
+	@Column(name="reservation_timeout")
+	LocalDateTime reservation_timeout;
+	@Column(name="booking_id")
+	String booking_id;
+	
+	public Ticket(Integer flight, Integer row, Character seat, Integer tier, Integer reserver, Integer price,
```

Comment:
Because these objects will typically not be created by our code, you'll probably need to define a default (no-argument) constructor for JPA and Jackson (the to-and-from-JSON library) to work properly, and this constructor will only ever get called from test code. I suggest just omitting it.